docs/serving/services/deployment.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Deployment Configuration"
+linkTitle: "Deployment Configuration"
+weight: 03
+type: "docs"
+---
+
+The `config-deployment` ConfigMap is located in the `knative-serving` namespace.
+This ConfigMap, known as the Deployment ConfigMap, contains settings that determine how Kubernetes `Deployment` resources, that back Knative services, are configured.
+
+## Accessing the Deployment ConfigMap 
+
+To view the current Deployment ConfigMap:
+
+```bash
+kubectl get configmap -n knative-serving config-deployment -oyaml
+```
+
+## Progress Deadline Seconds
+
+ProgressDeadline setting specifies the value of the `.spec.progressDeadlineSeconds` field on the underlying Kubernetes deployment for the Knative Revision.
+By default this value is set to 120 seconds.
+The value is expressed as a Go `time.Duration` string representation, but must be rounded to a second precision.
+Please see [Kubernetes documentation](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#progress-deadline-seconds) for details about this setting.
+
+In Knative the Autoscaler component gives up and scales the revision to 0 if the initial scale cannot be achieved within approximately this time span. If this happens the revision is considered to be failed.
```

Comment:
```suggestion
The Knative Autoscaler component scales the revision to 0 if the initial scale value cannot be achieved within the time limit defined by `.spec.progressDeadlineSeconds`. If this occurs, the revision deployment is considered to have failed.
```

Reply:
Not sure if this is accurate, but if not can you please clarify what "failing" means in this context? Is there some "failed" error message or status users can look at somewhere to figure out if the revision failed?

Reply:
So the `.spec.progressDeadlineSeconds` is not relevant here (it's deployment spec), here we should actually refer to the configmap setting we're describing how to set — since it's all in purview of knative.

Failing here means any problems that don't lead us to scaling to the initial scale.